### PR TITLE
Info subpage update

### PR DESCRIPTION
**Pull request by mjadach-iv**
- Info subpage update
- not turning on balance watcher if not connected
- https://github.com/hoprnet/hopr-admin/issues/158
- Ticket subpage

### CODE REVIEW

File: src/components/InfoBar/index.tsx [hunk 1]
```diff
@@ -4,12 +4,19 @@ import { useAppDispatch, useAppSelector } from '../../store';
...
 import styled from '@emotion/styled';
 import Toolbar from '@mui/material/Toolbar';
-import { ethers } from 'ethers';
+import { utils } from 'ethers';
 
 const drawerWidth = 160;
 
 interface Props {}
 
+const Contaier = styled(Box)`
```
Comment (camilovegag) [inline src/components/InfoBar/index.tsx:13]: small typo

File: src/future-hopr-lib-components/Navbar/navBar.tsx [hunk 2]
```diff
@@ -138,13 +138,13 @@ const NavBar: React.FC<{
               <MenuIcon />
             </IconButton>
             <Logo className="logo-hopr">
-              <a href="/">
+              {/* <a href="/"> */}
```
Comment (camilovegag) [inline src/future-hopr-lib-components/Navbar/navBar.tsx:141]: state was lost due to `a` tag, maybe a Link from react router would solve this?

File: src/components/InfoBar/index.tsx (hunk 1 above)
Comment (mjadach-iv) [reply src/components/InfoBar/index.tsx:13]: fixed

File: src/future-hopr-lib-components/Navbar/navBar.tsx (hunk 2 above)
Comment (mjadach-iv) [reply src/future-hopr-lib-components/Navbar/navBar.tsx:141]: We can do it

**Review by camilovegag — APPROVED** [review]

**Pull request merged**